src/components/Wizard/CreateVmWizard/StorageTab.js

```diff
@@ -425,7 +422,10 @@ class StorageTab extends React.Component {
       const values = {
```

Comment:
please could you refactor this so the if only mentions sourceType which are allowed?

Reply:
fixed